src/main/java/com/uid2/shared/secure/GcpOidcAttestationProvider.java

```diff
@@ -41,11 +41,15 @@ public void attest(byte[] attestationRequest, byte[] publicKey, Handler<AsyncRes
             if (enclaveId != null) {
                 handler.handle(Future.succeededFuture(new AttestationResult(publicKey, enclaveId)));
             } else {
-                throw new AttestationException("unauthorized token");
+                LOGGER.warn("Can not find registered gcp-oidc enclave id.");
```

Comment:
Can we mirror the logging in the AzureCC one - log the successful enclave id as an info and the unregistered one as a warn, with the enclaveId?